packages/marketplace/src/selector/__tests__/app-revision.ts

```diff
@@ -0,0 +1,48 @@
+import { ReduxState } from '@/types/core'
+import { revisionsDataStub } from '@/sagas/__stubs__/revisions'
+import { selectAppRevision, selectAppRevisions, selectAppRevisionDetail } from '../app-revisions'
+
+describe('app-revision', () => {
+  const mockState = {
```

Comment:
Same comment, would be nice to have a central state mock, then just override keys we need to change with `...rest`